tweb/src/main/frontend/src/js/vue/components/RequestListItem.vue

```diff
@@ -62,16 +63,23 @@
     export default class RequestListItem extends Vue {
         @Prop({required: true}) request!: any;
 
+        public requestItem: RequestDto = null;
+
+        mounted() {
+            this.requestItem = Object.assign(new RequestDto(), this.request);
+            this.requestItem.technologies = this.requestItem.technologies.sort();
+        }
+
         public get req() {
-            return Object.assign(new RequestDto(), this.request);
+            return this.requestItem;
         }
 
         public formatPrice(value, decimals: number = 2): string {
             return Utils.formatTokenPrice(value, decimals);
         }
 
         public showActions(event: Event) {
-            console.log('show actions');
+            console.log("show actions");
```

Comment:
Why do you need to log this?

Reply:
For the moment, this function is not used anymore. (It's for the extra actions in this list). But I updated it and commented it out for now.